manifests/1.3.7/opensearch-1.3.7.yml

```diff
@@ -10,13 +10,13 @@ ci:
 components:
   - name: OpenSearch
     repository: https://github.com/opensearch-project/OpenSearch.git
-    ref: db18a0d5a08b669fb900c00d81462e221f4438ee
+    ref: tags/1.3.7
     checks:
       - gradle:publish
       - gradle:properties:version
   - name: common-utils
     repository: https://github.com/opensearch-project/common-utils.git
-    ref: c350d4f243b817125b39b97de5b43fad45e8209d
+    ref: tags/1.3.7
```

Comment:
The tags for plugins should be 4 digits.

Reply:
Corrected!